lib/gooddata/bricks/middleware/logger_middleware.rb

```diff
@@ -14,14 +12,23 @@ class LoggerMiddleware < Bricks::Middleware
       def call(params)
         params = params.to_hash
         logger = nil
+
         if params['GDC_LOGGING_OFF']
           logger = NilLogger.new
         else
-          logger = params[:GDC_LOGGER_FILE].nil? ? Logger.new(STDOUT) : Logger.new(params[:GDC_LOGGER_FILE])
+          logger = params[:GDC_LOGGER_FILE].nil? ? GoodData::GDLogger.new(STDOUT) : GoodData::GDLogger.new(params[:GDC_LOGGER_FILE])
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)